gowrapper/cardano.go

```diff
@@ -62,22 +62,54 @@ func (addr *cardanoScriptConfig) toScriptConfig() *messages.CardanoScriptConfig
 	return nil
 }
 
+type cardanoToken struct {
+	*js.Object
+	AssetName []byte `js:"assetName"`
+	Value     string `js:"value"`
+}
+
+type cardanoAssetGroup struct {
+	*js.Object
+	PolicyID []byte          `js:"policyId"`
+	Tokens   []*cardanoToken `js:"tokens"`
+}
+
 type cardanoOutput struct {
 	*js.Object
 	EncodedAddress string               `js:"encodedAddress"`
 	Value          string               `js:"value"`
 	ScriptConfig   *cardanoScriptConfig `js:"scriptConfig"`
+	AssetGroups    []cardanoAssetGroup  `js:"assetGroups"`
 }
 
 func (output *cardanoOutput) toOutput() (*messages.CardanoSignTransactionRequest_Output, error) {
 	value, ok := new(big.Int).SetString(output.Value, 10)
 	if !ok {
 		return nil, errors.New("expected decimal string as value")
 	}
+	assetGroups := make([]*messages.CardanoSignTransactionRequest_AssetGroup, len(output.AssetGroups))
+	for i, assetGroup := range output.AssetGroups {
+		tokens := make([]*messages.CardanoSignTransactionRequest_AssetGroup_Token, len(assetGroup.Tokens))
+		for j, token := range assetGroup.Tokens {
+			tokenValue, ok := new(big.Int).SetString(token.Value, 10)
+			if !ok {
+				return nil, errors.New("expected decimal string as value")
+			}
+			tokens[j] = &messages.CardanoSignTransactionRequest_AssetGroup_Token{
```

Comment:
this is usually done with append, right? unless you want to optimize for speed for many many items:

```go
var groups []*messsages.CardanoSignTransactionRequest_AssetGroup
for _, g := range output.AssetGroups {
  var tokens []*messages.CardanoSignTransactionRequest_AssetGroup_Token
  for _, tok := range g.Tokens {
    ...
    tokens = append(tokens, &messages.CardanoSignTransactionRequest_AssetGroup_Token{
       ...
    }
  }
  groups = append(groups, &messages.CardanoSignTransactionRequest_AssetGroup{
    PolicyId: assetGroup.PolicyID,
    Tokens:   tokens,
  })
}
```

just reads nicer, more idiomatic.
or is there maybe some other gopherjs quirk?